packages/vite/package.json

```diff
@@ -138,7 +138,7 @@
     "sass": "*",
     "stylus": "*",
     "sugarss": "*",
-    "terser": "^5.4.0"
+    "terser": "^5.16.1"
```

Comment:
Bumping the terser peer dep version is a breaking change, I think we should probably check the terser version in the Vite code instead, before changing the target.

Reply:
Oh, I totally forgot about that. 😅
Changed to check the version 👍